presto-main/src/main/java/com/facebook/presto/cost/ComparisonStatsCalculator.java

```diff
@@ -57,7 +57,7 @@ private static PlanNodeStatsEstimate symbolToLiteralRangeComparison(PlanNodeStat
     {
         SymbolStatsEstimate symbolStats = inputStatistics.getSymbolStatistics(symbol);
 
-        StatisticRange range = new StatisticRange(symbolStats.getLowValue(), symbolStats.getHighValue(), symbolStats.getDistinctValuesCount());
+        StatisticRange range = StatisticRange.from(symbolStats);
```

Comment:
This whole commit should be done as set of retroactive fixups. Sorry but this is how we work on this branch :(